core/embed/trezorhal/stm32f4/touch/ft6x36.c

```diff
@@ -33,6 +33,12 @@
 #include "panels/lx154a2422cpt23.h"
 #endif
 
+#ifdef TOUCH_DETECT_ALL
+#define TOUCH_MAX_POINTS 2
```

Comment:
What's the reason for detecting all touch points in the prodtest?

Reply:
For detection of "ghost" touches, which are quite often two point. Just slightly increasing probability of these getting detected during prodtest.

Reply:
Then, maybe we could just use `touch_activity()` to detect unexpected behavior. With `touch_activity()` you can't report any additional information about the event, but it's not so important in the case. No sure, but I think it's better than adding such a strange configuration option to the touch driver.

Reply:
This works for `TOUCH_IDLE` test, but not for `TOUCH_CUSTOM` nor `TOUCH`, where we can also catch this via likely wrong coordinates in the ghost touch with two points. 

But i like it anyway, we will detect two points ghosts in TOUCH_IDLE and one point ghosts elsewhere, which seems acceptable https://github.com/trezor/trezor-firmware/pull/4064/commits/659dc9a74d12839070548f8b420862a9e382dcb5